src/modules/Packages.rb

```diff
@@ -1227,7 +1228,7 @@ def ComputeSystemPackageList
     # Import GPG keys found in the inst-sys
     def ImportGPGKeys
       out = Convert.to_map(
-        SCR.Execute(path(".target.bash_output"), "/bin/ls -d /*.gpg")
+        SCR.Execute(path(".target.bash_output"), "/usr/bin/ls -d /*.gpg")
```

Comment:
NP: maybe we could use some native Ruby here, as it reads from inst-sys we do not need any chroot magic.

Reply:
well, I do not where it is trivial enough, as I do not want to break it too much.